runconfig/parse.go

```diff
@@ -65,6 +65,7 @@ func Parse(cmd *flag.FlagSet, args []string) (*Config, *HostConfig, *flag.FlagSe
 		flCpuShares       = cmd.Int64([]string{"c", "-cpu-shares"}, 0, "CPU shares (relative weight)")
 		flCpusetCpus      = cmd.String([]string{"#-cpuset", "-cpuset-cpus"}, "", "CPUs in which to allow execution (0-3, 0,1)")
 		flCpusetMems      = cmd.String([]string{"-cpuset-mems"}, "", "MEMs in which to allow execution (0-3, 0,1)")
+		flBlkioWeight     = cmd.Int64([]string{"-blkio-weight"}, 0, "Block IO relative weight, range from 10 to 1000")
```

Comment:
Question; according to the docs;

>By default, all containers get the same proportion of block IO bandwidth (blkio). This proportion is 500.

But the default value here is `0`. Will this be confusing to people if they `docker inspect ...` the container? What does `docker inspect` show?